web/src/engine/websites/MangaPure.ts

```diff
@@ -0,0 +1,44 @@
+import { Tags } from '../Tags';
+import icon from './MangaPure.webp';
+import type { Manga } from '../providers/MangaPlugin';
+import { Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Madara from './decorators/WordPressMadara';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from '../FetchProvider';
+
+type MangaID = {
+    post: string,
+    slug: string
+}
+
+@Madara.MangaCSS(/^{origin}\/read\/[^/]+$/, 'div.post-title')
+@Madara.MangasMultiPageCSS(undefined, undefined, '/popular-manga?page={page}')
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangapure', 'MangaPure (manganelos)', 'https://mangapure.net', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.English, Tags.Source.Aggregator);
```

Comment:
Should there be a Bookmark migration from `manganelos` to `mangapure`?

Reply:
I asked myself but since its from AnyACG to Madara, maybe it just wont fit?